README.md

```diff
@@ -1,2 +1,85 @@
-# docker
-GeoServer docker image
+![alt text](./docker_hub_deployment.png?raw=true)
+
+# docker-geoserver
+
+## How to run it
+
+Pull the image from [Docker Hub](https://hub.docker.com/r/geosolutionsit/geoserver/)
+
+`docker pull geosolutionsit/geoserver`
+
+And run it
+
+`docker run --name gs -p 8080:8080 geosolutionsit/geoserver`
+
+Open your browser and point it to `http://localhost:8080/geoserver` .
+GeoServer web interface will show up, you can now log in with user admin and password `geoserver`.
+
+There are some [**environment variables**](https://docs.docker.com/engine/reference/run/) you can use at run time:
+- `JAVA_OPTS` to customize JAVA_OPTS for the container
+- `GEOSERVER_LOG_DIR` to customize log placement
+- `GEOSERVER_DATA_DIR` to put your GeoServer datadir elsewhere
+- `GEOWEBCACHE_CONFIG_DIR` to put your GeoServer cache configuration elsewhere
+- `GEOWEBCACHE_CACHE_DIR` to put your GeoServer cache elsewhere
+- `NETCDF_DATA_DIR` to put your GeoServer NETCDF data dir elsewhere
+- `GRIB_CACHE_DIR`o put your GeoServer GRIB cache dir elsewhere
+
+## How to build the Dockerfile with no helper scrips
+
+If you want to build the image by yourself just run `docker build` from the root of the repository
+
+` docker build -t geoserver:test . --build-args GEOSERVER_WEBAPP_SRC="./geoserver.war"`
+
+There are [**build arguments**](https://docs.docker.com/engine/reference/commandline/build/) to customize the image:
+- `GEOSERVER_DATA_DIR_SRC` to add your own custom datadir to the final image. This can be a local zip or directory or remote URL (see [ADD](https://docs.docker.com/engine/reference/builder/#add) instruction Doc)
+- `GEOSERVER_WEBAPP_SRC` to add your own custom web app to the final image. This can be a local zip or directory or remote URL (see [ADD](https://docs.docker.com/engine/reference/builder/#add) instruction Doc)
```

Comment:
seems that zip is not extracted automatically by ADD (but tar)

Reply:
This is handled by https://github.com/geoserver/docker/pull/2/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R17